climpred/comparisons.py

```diff
@@ -85,7 +86,7 @@ def get_comparison_function(comparison):
     elif comparison == 'm2r':
         comparison = _m2r
     else:
-        raise ValueError("""Please supply a comparison from the following list:
+        raise KeyError("""Please supply a comparison from the following list:
```

Comment:
or argument error. key error is even better

Reply:
I guess `KeyError` by python documentation description is technically for dictionary keys. But I also view it as a keyword. We can add `ArgumentError` if that's an issue.

Reply:
no key is fine. I made argument error up.